### PR TITLE
Avoid breaking docs site on specific searches

### PR DESCRIPTION
**Pull request by francocorreasosa**
Resolves https://github.com/auth0/cosmos/issues/592

### CODE REVIEW

**Comment by landitus** [comment]
Added a ref to the issue

**Comment by landitus** [comment]
Does this solve https://github.com/auth0/cosmos/issues/480 too? they loom quite similar

**Comment by nkohari** [comment]
Yep! Good catch.

File: src/docs/sidebar/children.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const attachChildren = components => {
 
   children.forEach(child => {
     const parent = getParent(parents, child)
-    parent.children.push(child)
+    if (parent) parent.children.push(child)
```
Comment (siddharthkp) [inline src/docs/sidebar/children.js]: 🙈👍👍

**Pull request merged**